batch_processing.sh

```diff
@@ -259,12 +259,10 @@ echo "Version:         `sct_version`"
 echo "Ran on:          `uname -nsr`"
 echo "Duration:        $(($runtime / 3600))hrs $((($runtime / 60) % 60))min $(($runtime % 60))sec"
 echo "---"
-echo "t2/CSA:         " `awk -F"," ' {print $6}' t2/csa_c2c3.csv | tail -1`
-echo "mt/MTR(WM):     " `awk -F"," ' {print $8}' mt/mtr_in_wm.csv | tail -1`
-echo "t2s/CSA_GM:     " `awk -F"," ' {print $6}' t2s/csa_gm.csv | tail -1`
-echo "t2s/CSA_WM:     " `awk -F"," ' {print $6}' t2s/csa_wm.csv | tail -1`
-echo "dmri/FA(CST_r): " `awk -F"," ' {print $8}' dmri/fa_in_cst.csv | tail -1`
-echo "dmri/FA(CST_l): " `awk -F"," ' {print $8}' dmri/fa_in_cst.csv | tail -2 | head -1`
+# The file `test_batch_processing.py` will output tested values when run as a script
+"$SCT_DIR"/python/envs/venv_sct/bin/python $SCT_DIR/unit_testing/batch_processing/test_batch_processing.py
+# If you want to actually test these values against SCT's ground truth, run the following line as well:
+# TEST_BATCH_PROCESSING=1 "$SCT_DIR"/python/envs/venv_sct/bin/python -m pytest "$SCT_DIR/unit_testing/batch_processing/test_batch_processing.py"
```

Comment:
I'm wondering if we might want to run the test for users of `batch_processing.sh` by default. Hm.

Reply:
The only downside to this is that right now, the test data downloading functionality is kind of broken -- it downloads the test data to the current working directory every time `pytest` is run. I'm looking to fix that in the future as I continue to rework testing data (#3422), but until then, I think it might be best to leave this commented out.

Reply:
Hm... good question. I would lean towards a yes, so that if a user has an issue they can copy/paste their terminal without the need to re-run the entire batch_processing (which is relatively long to run-- ~10min)

Reply:
Some related discussions for cross-linking:

* https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3310#issuecomment-805972981
* https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3312

Reply:
Actually, now that I think about it, https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3194 will probably be a thorn in our side, too. (Reports from macOS users that the test failed, etc.)

I think based on that I'm going to remove these lines entirely.